internal/scaffold/olm-catalog/csv_updaters.go

```diff
@@ -62,8 +62,8 @@ func NewUpdaterStore() *updaterStore {
 }
 
 // Apply iteratively calls each stored CSVUpdater's Apply() method.
-func (s *updaterStore) Apply(csv *olmapiv1alpha1.ClusterServiceVersion) error {
-	updaters := []CSVUpdater{s.installStrategy, s.crds, s.almExamples}
+func (store *UpdaterStore) Apply(csv *olmapiv1alpha1.ClusterServiceVersion) error {
+	updaters := []CSVUpdater{store.installStrategy, store.crds, store.almExamples}
```

Comment:
What is the linter complaining about here? -1 I think `s` is just fine. I use single letters for all of my struct receivers. I agree `store` is easier to read, but I knew it was the store because of the receiver. It's a very common paradigm. In our broker repo there are 12260 instances of `func (a *`

Reply:
Nevermind, I see it now. You are simply making it consistent.

Reply:
@joelanford I know that your only concern with this one still this point. However, I do see it broken the interface since it is used outside of the file shows that could not be the lower case at all. 

So, if you agree and it is ok for you too. I think we could move forward this one and get it merged. WDYT?